app/helpers/application_helper/toolbar/miq_ae_tools_simulate_center.rb

```diff
@@ -6,6 +6,7 @@ class ApplicationHelper::Toolbar::MiqAeToolsSimulateCenter < ApplicationHelper::
       N_('Copy object details for use in a Button'),
       N_('Copy'),
       :url       => "resolve",
-      :url_parms => "?button=copy"),
+      :url_parms => "?button=copy",
+      :klass     => ApplicationHelper::Button::ButtonWithoutRbacCheck),
```

Comment:
@romanblanco There should be hover text on that button to tell the user that they can copy all of these options, then paste them into another area, which I "think" is maybe the creation of a button or perhaps somewhere in Automate when creating a method.  @h-kataria do you recall where the related "Paste" button(s) is/are?